include/sparrow/data_traits.hpp

```diff
@@ -55,6 +55,12 @@ namespace sparrow
         static constexpr data_type type_id = data_type::INT8;
     };
 
+    template <>
+    struct arrow_traits<char> : common_native_types_traits<char>
+    {
+        static constexpr data_type type_id = data_type::INT8;
```

Comment:
Shouldnt it be `UINT8`? Note that `char` is not considered signed so maybe either one is fine.

Reply:
Modified